docs/standard/linq/toc.yml

```
@@ -0,0 +1,346 @@
+- name: LINQ Overview.md
+  href: linq-overview.md
+- name: LINQ to XML
+  items:    
+  - name: Getting Started (LINQ to XML)
+    items:
+    - name: LINQ to XML Overview
+      href: linq-to-xml-overview.md
+    - name: LINQ to XML vs. DOM
+      href: linq-to-xml-vs-dom.md
+    - name: LINQ to XML vs. Other XML Technologies
+      href: linq-to-xml-vs-other-xml-technologies.md
+  - name: Programming Guide (LINQ to XML)
+    items:
+    - name: LINQ to XML Programming Overview
+      items:
+      - name: Functional vs. Procedural Programming (LINQ to XML)
+        href: functional-vs-procedural-programming-linq-to-xml.md
```

I think we could remove the linq-to-xml suffix from these file names 

```suggestion
        href: functional-vs-procedural-programming.md
```